governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Recentering Maker Brand - November 4, 2024
+summary: Signal your support or opposition to 1) keep the Sky brand as the backend protocol brand; or 2) recenter the Maker brand; or 3) recenter the Maker brand with a limited brand refresh.
```

Comment:
Support or opposition is not accurate, action required. Suggestion:

Signal your support of the following branding options to...